{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:python:setuptools:58.0.3:*:*:*:*:*:*:*",
	"cpeNameId": "4AC39732-17F5-4D7E-9E87-E24F44957AD9",
	"created": "2022-12-29T14:39:02.883",
	"lastModified": "2022-12-29T14:43:10.977",
	"titles": [
		{
			"title": "Python setuptools 58.0.3",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://pypi.org/project/setuptools/#history",
			"type": "Version"
		},
		{
			"ref": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/",
			"type": "Advisory"
		}
	]
}
